code/datums/elements/plasma_on_attack.dm

```diff
@@ -0,0 +1,17 @@
+/datum/element/plasma_on_attack
+	element_flags = ELEMENT_BESPOKE
+	id_arg_index = 2
+
+	var/damage_plasma_multiplier = 1
+
+/datum/element/plasma_on_attack/Attach(datum/target, damage_plasma_multiplier)
+	. = ..()
+	if(!isxeno(target))
+		return ELEMENT_INCOMPATIBLE
+	RegisterSignal(target, COMSIG_XENOMORPH_ATTACK_LIVING, .proc/damage_dealt)
+	src.damage_plasma_multiplier = damage_plasma_multiplier
+
+/datum/element/plasma_on_attack/proc/damage_dealt(datum/source, mob/living/attacked, damage)
```

Comment:
autodoc

Reply:
don't think i need to autodoc this since its a very small proc and what it does is pretty apparent

Reply:
The rule is autodoccing everything though

Reply:
also signal handler missing